src/curated-corpus/components/ApprovedItemSearchForm/ApprovedItemSearchForm.test.tsx

```diff
@@ -86,9 +102,35 @@ describe('The CuratedItemSearchForm component', () => {
     userEvent.clear(urlField);
     userEvent.type(urlField, 'abc');
     await waitFor(() => {
-      userEvent.click(button);
+      userEvent.click(searchButton);
     });
 
     expect(handleSubmit).toHaveBeenCalled();
   });
+
+  it('resets all form filters when Reset Filters button is clicked', async () => {
+    render(<ApprovedItemSearchForm onSubmit={handleSubmit} />);
+
+    // get text input fields
+    const titleField = screen.getByLabelText(/filter by title/i);
+    const urlField = screen.getByLabelText(/filter by url/i);
```

Comment:
You can see that I'm only getting two input fields here where as we have two drop downs. I have omitted them for now since react-testing library and Formik were giving me a tough time. The screen cap covers the scenario but I can bolster this test in a follow up PR

Reply:
A follow-up PR sounds good, I know the dropdowns can be tricky to test. I have had some luck with the `getByDisplayValue` method from React Testing Library, ([see example](https://github.com/Pocket/curation-admin-tools/blob/c41bffed2c8e25fce2e0023722d6900ad10fa778/src/curated-corpus/components/ApprovedItemForm/ApprovedItemForm.test.tsx#L79)).